helm/csi-isilon/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 name: csi-isilon
-version: 2.4.0
-appVersion: 2.4.0
-kubeVersion: ">= 1.21.0 < 1.25.0"
+version: 2.5.0
+appVersion: 2.5.0
+kubeVersion: ">= 1.21.0 <= 1.25.2"
```

Comment:
max kube verson supported should be 1.25.x . Check should be < 1.26.0 instead

Reply:
changed it